### ISSUE TITLE
Error while defining enum comments

### ISSUE THREAD

**Issue by fahernandez** [grpc-ecosystem/grpc-gateway#897]
### Steps to reproduce the issue

**Steps**
1.  Define an enum proto message with comments containing a **.**
```
// DeepEnum is one or zero.
enum DeepEnum {
    // FALSE is false.
    FALSE = 0;
   // TRUE is true.
   TRUE = 1;
}

// DeepEnum comment.<--This dot
DeepEnum ok = 3 [(grpc.gateway.protoc_gen_swagger.options.openapiv2_field) = {description: "DeepEnum description."}];
```
2. Generate the swagger documentation.

**Expected behavior**
1. The swagger code is generated without errors

**Actual behavior**
1. An error code is triggered.

**Hipotesis**
This error is triggered by an error on protoc-gen-swagger/genswagger/template.go:1268. The solution is to refactor the **if** behavior.

If you remove the dot from the comment it will generate the swagger code without errors.

**Comment by johanbrandhorst** [comment]
What does the generated swagger look like? Would you like to contribute a fix for this?

**Comment by fahernandez** [comment]
The swagger file is not generated due the error code. Sure I will fix it.

**Comment by johanbrandhorst** [comment]
Fixed by #898
